lib/actions.js

```diff
@@ -0,0 +1,6 @@
+function fetchPlants() {
+  return fetch(`http://harvesthelper.herokuapp.com/api/v1/plants?api_key=${"api key"}`).then(response => response.json().then(json => {
+    debugger;
```

Comment:
good catch.  I am not using this file yet but will take it out when I do and start manipulating data.